Makefile

```
@@ -24,10 +24,13 @@ ifeq (, $(shell which docker))
 endif
 
 #### Development ####
-# start commands
+# Builds a vagrant box (included Ansible playbooks) for example/standalone, without running the tests.
 dev-standalone: update-box custom_ca
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--skip-tags "test" --extra-vars "\"mode=standalone\""' vagrant up --provision
 
+# Builds a vagrant box (included Ansible playbooks) for example/standalone_git, without running the tests.
+# Remember the parameters for GIT integration explained in /example/standalone_git/README.md
+# Example: make dev repo=<GitHub-repository> branch=<branch to checkout and track> user=<GitHub username> token=<personal token from GitHub>
 dev: update-box custom_ca
```

I guess you need to add `check-params` step here (since it runs example/standalone_git):

```suggestion
dev: check-params update-box custom_ca
```

---

Thanks, i see that i forgot to add params as well

---

Done